docker-compose.yml

```diff
@@ -161,21 +161,23 @@ services:
       KAFKA_LDAP_JAVA_NAMING_FACTORY_INITIAL: com.sun.jndi.ldap.LdapCtxFactory
       KAFKA_LDAP_COM_SUN_JNDI_LDAP_READ_TIMEOUT: 3000
       KAFKA_LDAP_JAVA_NAMING_PROVIDER_URL: ldap://openldap:389
-      # How to authenticate to LDAP
+      # Authenticate to LDAP
```

Comment:
I saw you mentioned AD/LDAP in this file. Is this setup using AD or OpenLdap?

Reply:
@jeffhuang26 OpenLDAP: https://github.com/confluentinc/cp-demo/blob/5.4.1-post/docker-compose.yml#L34